docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -1,500 +1,607 @@
-  Logged in as:       myuser
-  Cluster:            rootcluster.teleport.sh
-  Roles:              access, auditor, editor, reviewer
-  Logins:             root
-  Kubernetes:         enabled
-  Valid until:        2022-04-29 03:07:22 -0400 EDT [valid for 12h0m0s]
-  Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty
-```
+- The list of **principals** that the user is authorized to use. Principals are equivalent to
+  the local logins that have been added to a Teleport user profile.
+- The signature of the **certificate authority** that issued the certificate. In a trusted cluster environment, 
+  the Teleport Auth Service on the root cluster signs the certificate.
+- The **metadata certificate extensions** that are provided by the Teleport Auth Service. Teleport uses 
+  the metadata to store the list of user roles and SSH options like `permit-agent-forwarding`.
+- The expiration date, or **time-to-live** (TTL) to ensure the certificate is still valid.
 
-</TabItem>
+Based on the information in the certificate, the Teleport Auth Service performs the following actions:
 
-</Tabs>
+- Verifies that the certificate signature matches one of its trusted root clusters.
+- Applies role mapping to associate a role on the leaf cluster with one of the roles assigned 
+  to the user in the root cluster.
+- Checks if the local role allows the requested identity—the Unix login—to have access.
+- Verifies that the certificate has not expired. The TTL is set by the root cluster.
 
-Execute the following command on your development machine:
+The following diagram provides a simplified view of the interaction between the services running
+on the leaf cluster and service running on the root cluster:
 
-```code
-# Generates a Trusted Cluster token to allow an inbound connection from a leaf cluster:
-$ tctl tokens add --type=trusted_cluster --ttl=5m
-The cluster invite token: (=presets.tokens.first=)
-This token will expire in 5 minutes
+![Service interaction in a trusted cluster](../../../img/trusted-clusters/trusted-cluster-service-interaction.png)
 
-Use this token when defining a trusted cluster resource on a remote cluster.
-```
 
-This command generates a Trusted Cluster join token. The token can be used
-multiple times and has an expiration time of 5 minutes.
+Note that trusted clusters only work in one direction. 
+Users from the leaf cluster can't see or connect to resources in the root cluster.
 
-<Details title="Optional: Add labels">
-The `--labels` flag will add labels to the trusted cluster. This lets teams use
-the same RBAC controls used on individual resources to approve or deny access
-to clusters.
-</Details>
+## Role relationships in a trusted cluster
 
-Copy the join token for later use. If you need to display your join token again,
-run the following command against your root cluster:
+As noted in [How it works](#how-it-works), leaf clusters are autonomous. They have their own state, roles, 
```

Comment:
Do we need to re-explain this information here if it's already explained in a previous section in the same page?

Reply:
Just a transition device.